sql/log_event.cc

```diff
@@ -4972,6 +4972,20 @@ int Query_log_event::do_apply_event(Relay_log_info const *rli,
       clear_all_errors(thd, const_cast<Relay_log_info*>(rli));
       thd->killed= THD::NOT_KILLED;
     }
+    /*
+      Tianmu engine as slave: ingnor some errors
+    */    
+    else if(ER_TIANMU_NOT_SUPPORTED_SECONDARY_INDEX == actual_error
+            || ER_TIANMU_NOT_SUPPORTED_UNIQUE_INDEX  == actual_error
+            || ER_TIANMU_NOT_SUPPORTED_FULLTEXT_INDEX  == actual_error
+            || ER_TIANMU_NOT_SUPPORTED_TRIGGER == actual_error
+            || ER_TIANMU_NOT_SUPPORTED_FOREIGN_KEY == actual_error
```

Comment:
there are 10 kinds of erros in sql layer, why only filter 6 kinds of errors? 
```
#define ER_TIANMU_NOT_SUPPORTED_SECONDARY_INDEX 3234
#define ER_TIANMU_NOT_SUPPORTED_UNIQUE_INDEX 3235
#define ER_TIANMU_NOT_SUPPORTED_FULLTEXT_INDEX 3236
#define ER_TIANMU_NOT_SUPPORTED_GEOMETRY 3237
#define ER_TIANMU_NOT_SUPPORTED_ENUM 3238
#define ER_TIANMU_NOT_SUPPORTED_SET 3239
#define ER_TIANMU_NOT_SUPPORTED_TRIGGER 3240
#define ER_TIANMU_NOT_SUPPORTED_FOREIGN_KEY 3241
#define ER_TIANMU_NOT_SUPPORTED_PARTITION 3242
#define ER_TIANMU_NOT_FOUND_INDEX 3243
```

Reply:
Because the product department has adjusted it‘s requirements, it does not need support in 1.0.3 `data type, spatial SPATIAL index and partition`